src/tools/cephfs/top/cephfs-top

```diff
@@ -292,7 +296,14 @@ if __name__ == '__main__':
                         help='Path to cluster configuration file')
     parser.add_argument('--selftest', dest='selftest', action='store_true',
                         help='run in selftest mode')
+    parser.add_argument('-d','--delay', nargs='?', default=REFRESH_INTERVAL,
+                        type=float,help='Interval to refresh data (defualt: 1)')
```

Comment:
```py
def float_greater_than(min):
    def check(s):
      value = float(s)
      value < min:
        raise argparse.ArgumentTypeError(f"{value} should be greater than {min}")
      return value
# ...
parser.add_argument('-d', '--delay', nargs='?', default=REFRESH_INTERVAL,
                    type=float_greater_than(MIN_REFRESH_INTERVAL),
                    help='Interval to refresh data (default: 1)')
```

couple things

- might want to add a customized `type` for better error message
- add spaces before comma
- fix typo